google-cloud-spanner/src/test/java/com/google/cloud/spanner/ValueBinderTest.java

```diff
@@ -109,17 +111,32 @@ public void reflection() throws InvocationTargetException, IllegalAccessExceptio
       } else if (binderMethod.getParameterTypes().length == 1) {
         // Test unary null.
         if (!binderMethod.getParameterTypes()[0].isPrimitive()) {
+          if (method.getName().toLowerCase().equals("json")) {
```

Comment:
nit: Can you extract a constant for `"json"`? Maybe `JSON_METHOD_NAME` or something alike?